onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -37,6 +37,7 @@
 #include "core/optimizer/initializer.h"
 #include "core/optimizer/layer_norm_fusion.h"
 #include "core/optimizer/matmul_add_fusion.h"
+#include "core/optimizer/matmul_scale_fusion.h"
```

Comment:
Would matmul_scale fusion conflict with matmul_transpose fusion? 
Does the order matters which one is applied first? 
I hope both of them handle the case of TranposeMatmul+otherOp => TranposeMatmul

Reply:
I think the order shouldn't matter, as the transformers will be iteratively applied while they continue to change the graph.
It looks like both MatmulTransposeFusion and MatMulScaleFusion handle both MatMul and TransposeScaleMatMul as the base node.

Reply:
good to hear that. 👍